src/views/CreatePage.vue

```diff
@@ -304,6 +319,22 @@ div(class="flex justify-center items-center my-4")
   button(v-show="saveState === 'done'" type="button" class="flex items-center focus:outline-none text-white bg-seaPink hover:bg-red-400 focus:ring-4 focus:ring-red-300 font-medium rounded-lg px-4 py-1.5")
     span(class="material-symbols-outlined") done
 
+  div(class="relative ml-8")
+    button(class="three-dot-leader hover:opacity-80" type="button" @click="pointLeaderOpen = !pointLeaderOpen")
+        span(class="dot")
+    button(v-show="pointLeaderOpen" tabindex="-1" class="z-10 fixed inset-0 h-full w-full cursor-default" @click="pointLeaderOpen = false")
+    div(v-show="pointLeaderOpen"  class="absolute z-10 mt-2 py-2 w-max bg-white rounded-lg shadow-xl")
+      div(class="flex items-center cursor-auto px-4 py-2 text-gray-800") 
+        span(class="material-symbols-outlined mr-2" style='color:#2F90EA') open_in_new
+        div(class="mr-2 text-lg") 共有
+        label(class="relative inline-flex items-center cursor-pointer")
+          input(:checked="isShare" type="checkbox" class="sr-only peer" @click="saveIsShare()")
```

Comment:
```suggestion
          input(:checked="isShare" type="checkbox" class="sr-only peer" @click="saveIsShare")
```
細かいですが、()で呼び出さなくてもよかった気がします。